css/src/atomics/width.scss

```diff
@@ -0,0 +1,101 @@
+$widths: 100, 150, 200, 250, 300, 350, 400, 450, 500 !default;
```

Comment:
I think `500` can be removed. At first, I couldn't find any existing case using that width. Secondly, not providing that option would eliminate the number of problems with mobile screens. Even though I understand some others still might be an issue, but at least just one less 🤔

Reply:
perhaps we should eliminate 450 too?

Reply:
Perhaps 500 can be added once there is a use case. I see that we're currently using 450 for the chat window in LearnTV so we might want to keep that.
![image](https://user-images.githubusercontent.com/94572161/149564376-6e08894f-ad08-47aa-9f41-290c9c352297.png)

Reply:
As said in other thread, re:LearnTV example it seems to me that particular instance could be either removed or changed to a slightly lower value or only applied on desktop.

Reply:
Was a WI created to make that update in docs? Resolve if it was or after creating one!

Reply:
done!